xesmf/frontend.py

```diff
@@ -12,6 +12,21 @@
 from . smm import read_weights, apply_weights
 
 
+def get_latlon_name(ds):
+    # COARDS netCDF complaint
+    try:
+      if 'lat' in ds.variables:
+        lat_name = 'lat'
+        lon_name = 'lon'
+    # NETCDF CF 1.6 complaint 
+      elif 'latitude' in ds.variables:
+        lat_name = 'latitude'
+        lon_name = 'longitude'
+      else:
+        raise ValueError
+     except ValueError:
+        print('Must have coordinates compliant with NETCDF COARDS or CF conventions')
+
```

Comment:
As written this doesn't function doesn't return anything.  Needs something like 
```suggestion
    return lon_name, lat_name
```